src/aat/resources/features/F-058/S-517.td.json

```diff
@@ -0,0 +1,24 @@
+{
+  "_guid_": "S-517",
+  "_extends_": "F-058_Test_Data_Base",
+
+  "title": "must return 404 when request contains a non-existing jurisdiction ID",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains a non-existing jurisdiction ID",
+    "code is HTTP-403"
+  ],
+  "request": {
+    "pathVariables": {
+      "jid": "Dummy"
```

Comment:
please use arbitrary value which won't clash. may be something like 'eyJhbGciOiJIUzUxMiJ9'